ERROR - 2023-03-22 12:26:35 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:26:35 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-22 12:26:36 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:26:36 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-22 12:26:36 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:26:36 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-22 12:26:42 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:26:42 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-22 12:26:56 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:26:56 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-22 12:27:10 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bd_taller', 3306, '', 0)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#4 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#10 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xamppp\htdocs\taller\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xamppp\htdocs\taller\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xamppp\htdocs\taller\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`...', Array, false)
#3 C:\xamppp\htdocs\taller\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xamppp\htdocs\taller\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xamppp\htdocs\taller\app\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xamppp\htdocs\taller\app\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
#7 C:\xamppp\htdocs\taller\system\CodeIgniter.php(934): App\Controllers\Cargos->index()
#8 C:\xamppp\htdocs\taller\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
#9 C:\xamppp\htdocs\taller\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xamppp\htdocs\taller\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-22 12:27:10 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cargos`.*
FROM `cargos`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\CargosModel.php(38): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Cargos.php(17): App\Models\CargosModel->obtenerCargos('A')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Cargos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cargos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
